ruby_event_store/spec/serialized_record_spec.rb

```diff
@@ -0,0 +1,19 @@
+module RubyEventStore
+  module Mappers
+    RSpec.describe SerializedRecord do
+      let(:id)         { double(:id) }
+      let(:data)       { double(:data) }
+      let(:metadata)   { double(:metadata) }
+      let(:event_type) { double(:event_type) }
+
+      specify '#event_to_serialized_record returns YAML serialized record' do
```

Comment:
`'#event_to_serialized_record returns YAML serialized record'` nope, not really what the test is about.